### PR TITLE
feat(react): new automatic runtime

### PR DESCRIPTION
**Pull request by philibea**
It's will possibly improve the size on all bundle using react, but we will be more coherent with all ours projects

### CODE REVIEW

**Comment by QuiiBz** [comment]
Should we disable the `react/react-in-jsx-scope` rule? ESLint don't like it: 

https://github.com/scaleway/scaleway-lib/runs/5787900573?check_suite_focus=true#step:7:12

**Comment by philibea** [comment]
> Should we disable the `react/react-in-jsx-scope` rule? ESLint don't like it:
> 
> https://github.com/scaleway/scaleway-lib/runs/5787900573?check_suite_focus=true#step:7:12

yes exactly, it's already done

**Review by QuiiBz — APPROVED** [review]

File: packages/use-segment/src/__tests__/index.tsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { AnalyticsBrowser, Context } from '@segment/analytics-next'
 import { renderHook } from '@testing-library/react-hooks'
-import React, { ReactNode } from 'react'
+import { ReactNode } from 'react';
```
Comment (chambo-e) [inline packages/use-segment/src/__tests__/index.tsx:3]: intended ?

Comment (philibea) [reply packages/use-segment/src/__tests__/index.tsx]: fixed with the format https://github.com/scaleway/scaleway-lib/pull/696

**Review by chambo-e — APPROVED** [review]

**Pull request merged**